agent_build_refactored/managed_packages/convenience_install_script/install_agent_template.sh

```diff
@@ -0,0 +1,432 @@
+#!/bin/bash
```

Comment:
This is basically just a copy of [existing install script ](https://github.com/scalyr/scalyr-agent-2/blob/master/scripts/circleci/release/installScalyrAgentV2.sh) but without some legacy parts.